plugins/operators/__init__.py

```diff
@@ -1442,40 +1449,56 @@ def config(self):
     def resolve_input(self, ctx):
         inputs = types.Object()
 
-        summary_fields = ctx.dataset.list_summary_fields()
-
-        if summary_fields:
-            field_selector = types.AutocompleteView()
-            for key in summary_fields:
-                field_selector.add_choice(key, label=key)
-
-            inputs.enum(
-                "field_name",
-                field_selector.values(),
-                required=True,
-                label="Summary field",
-                description="The summary field to delete",
-                view=field_selector,
-            )
-        else:
-            prop = inputs.str(
-                "field_name",
-                label="This dataset does not have summary fields",
-                view=types.Warning(),
-            )
-            prop.invalid = True
+        _delete_summary_field_inputs(ctx, inputs)
 
         return types.Property(
             inputs, view=types.View(label="Delete summary field")
         )
 
     def execute(self, ctx):
-        field_name = ctx.params["field_name"]
+        field_names = _to_string_list(ctx.params["field_names"])
 
-        ctx.dataset.delete_summary_field(field_name)
+        ctx.dataset.delete_summary_fields(field_names)
         ctx.trigger("reload_dataset")
 
 
+def _delete_summary_field_inputs(ctx, inputs):
+    summary_fields = ctx.dataset.list_summary_fields()
+
+    if not summary_fields:
+        prop = inputs.str(
+            "field_name",
+            label="This dataset does not have summary fields",
+            view=types.Warning(),
+        )
+        prop.invalid = True
+        return
+
+    field_names = _to_string_list(ctx.params.get("field_names", []))
+
+    field_selector = types.AutocompleteView(multiple=True)
```

Comment:
Ideally can make this `DropdownView(multiple=True)` to avoid manually enforcing valid values below